hack/jenkins/build.sh

```diff
@@ -62,7 +62,11 @@ git clean -fdx
 # docker images -q | xargs -r docker rmi
 
 # Build
-go run ./hack/e2e.go -v --build
+if [[ "${KUBE_FASTBUILD:-}" == "true" ]]; then
```

Comment:
I think we just want to make this `make release` - it should the right thing when `KUBE_FASTBUILD` is set (which we do for PR builds, for example).

(I also feel like maybe we should remove the `--build` option from `hack/e2e.go`...)